src/layman/util.py

```diff
@@ -49,6 +52,13 @@ def to_safe_names(unsafe_names, type_name):
     return values
 
 
+def check_deprecated_url(response):
+    if not request.path.startswith('/rest/workspaces/'):
```

Comment:
I suggest to match against regexp `^/rest/workspaces\b.*$` to cover also (potential future) requests to `/rest/workspaces` endpoint